app/controllers/account/admin/categories_controller.rb

```diff
@@ -0,0 +1,94 @@
+# Encoding: utf-8
+module Account
+  module Admin
+    class CategoriesController < ApplicationController
+      before_action :admin_or_moderator?
+      before_action :status_inspector, only: [:show]
+      before_action :find_category, except: [:index, :create, :new]
+
+      def index
+        @categories = Category.all
+        smart_listing_create :categories, @categories,
+                             sort_attributes: [
+                               [:created_at, 'categories.created_at'],
+                               [:updated_at, 'categories.updated_at'],
+                               [:name, 'categories.name'],
+                               [:calculate, 'categories.issues_count']
+                             ],
+                             default_sort: { name: 'asc' }, partial: 'category'
+      end
+
+      def show
+        @status = params[:status] || 'opened'
+        issues_scope = @category.issues.ordered
+                                .where(status: @status)
+        issues_scope = issues_scope.like(params[:filter]) if params[:filter]
+        smart_listing_create :issues,
+                             issues_scope,
+                             partial: 'issues/issue'
+        @categories = Category.ordered_by_name
+        render 'issues/index'
+      end
+
+      def new
+        @category = Category.new
+      end
+
+      def create
+        @category = Category.new(category_params)
+        if @category.save
+          flash[:success] = 'Категорію створено!'
+          redirect_to account_admin_categories_path
+        else
+          render 'new'
+        end
+      end
+
+      def edit; end
+
+      def update
+        @category.assign_attributes(category_params)
+        flash[:notice] = 'Ви не внесли жодних змін' unless @category.changed?
```

Comment:
@denys281, I think it's really incorrect to change value of "updated_at" field, if there wasn't any changes made.